godtools/App/Services/Renderer/Parser/Multiplatform/Models/MultiplatformContentButton.swift

```diff
@@ -80,7 +80,7 @@ class MultiplatformContentButton: ContentButtonModelType {
     }
     
     var textScale: MobileContentTextScale {
-        return MobileContentTextScale(doubleValue: button.textScale)
+        return MobileContentTextScale(doubleValue: button.text?.textScale ?? 1.0)
```

Comment:
```suggestion
        return MobileContentTextScale(doubleValue: button.text?.textScale ?? button.textScale)
```
it's probably safer to just default back to the text scale that the button inherits instead of to a constant of 1.0

Reply:
Hey @reldredge71 can you hold on this PR until  GT-1422 is merged?  MobileContentTextScale is going to be non existent once that is merged in.  And GT-1422 is ready for review now :)

Reply:
@levieggertcru Sounds good.  I'll look through GT-1422 now.